test/fixtures/mson/enum-multiple-default.json

```diff
@@ -0,0 +1,315 @@
+{
+  "element": "parseResult",
+  "content": [
+    {
+      "element": "category",
+      "meta": {
+        "classes": {
+          "element": "array",
+          "content": [
+            {
+              "element": "string",
+              "content": "api"
+            }
+          ]
+        },
+        "title": {
+          "element": "string",
+          "content": ""
+        }
+      },
+      "content": [
+        {
+          "element": "category",
+          "meta": {
+            "classes": {
+              "element": "array",
+              "content": [
+                {
+                  "element": "string",
+                  "content": "dataStructures"
+                }
+              ]
+            }
+          },
+          "content": [
+            {
+              "element": "dataStructure",
+              "content": {
+                "element": "object",
+                "meta": {
+                  "id": {
+                    "element": "string",
+                    "content": "Enum behavior"
+                  }
+                },
+                "content": [
+                  {
+                    "element": "member",
+                    "content": {
+                      "key": {
+                        "element": "string",
+                        "content": "e1"
+                      },
+                      "value": {
+                        "element": "enum",
+                        "attributes": {
+                          "default": {
+                            "element": "enum",
+                            "content": {
+                              "element": "string",
+                              "content": "K"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  },
+                  {
+                    "element": "member",
+                    "content": {
+                      "key": {
+                        "element": "string",
+                        "content": "e2"
+                      },
+                      "value": {
+                        "element": "enum",
+                        "attributes": {
+                          "default": {
+                            "element": "enum",
+                            "content": {
+                              "element": "string",
+                              "content": "K"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  },
+                  {
+                    "element": "member",
+                    "content": {
+                      "key": {
+                        "element": "string",
+                        "content": "e3"
+                      },
+                      "value": {
+                        "element": "enum",
+                        "attributes": {
```

Comment:
Are `enumerations` missing here?

Reply:
There are no enumerations defined in the MSON line: `- e3: a,b,c (enum, default)`. Since there is the `default` type attribute, all a, b, and c are treated as the default value(s). Enum doesn't support multiple defaults so there is a warning annotation below: 

> multiple definitions of 'default' value

Reply:
According to the [MSON Spec](https://github.com/apiaryio/mson/blob/master/MSON%20Specification.md#212-structure-types) an `enum`
> Specifies an exclusive list of possible Values or Types for values.

From my understanding, this list is what we put into our `enumerations` attribute.

So if the value of the member element in `- e3: a,b,c (enum, default)` is parsed as an MSON enum, it is _defined_ by an exclusive list of its possible MSON Values or MSON Types. In this case, I noticed we don't error out, so I expect this list is well defined. What is it in this case? `[a,b,c]`? `[]`?

The idea of leaving out the `enumerations` attribute is limited to `samples` and `default` _inside_ the definition of an enum element, because the `enumerations` of a value always match the `enumerations` of its type.